### DISCUSSION TITLE
v15 on secured Mac

### DISCUSSION THREAD

**Discussion by sanimalp** [pqrs-org/Karabiner-Elements#3996] [Q&A]
I have been using KE on my work mac for 6 or so years now, in various locked down environments. Version 15 installation seems to require the addition of "login items", and abruptly stops working upon installation. I can revert back to version 14.13.0 and all my simple and complex remappings work as normal, but as soon as my system auto-updates to v15, they break again.

Is there a way to be on the latest version but not require the login-items entries my IT group has informed me under no circumstances will i be granted? I am afraid without it, this constant revert to old version cycle will have to continue. 

Maybe I have also missed a simple configuration to make v15 work in this sort of environment... any help is appreciated.

**Comment by tekezo** [comment]
I've modified it to run the necessary processes via login items, so it generally cannot be used unless this is enabled.

While it might work if you manually run the program specified in the plist under `/Library/Application Support/org.pqrs/Karabiner-Elements/Karabiner-Elements Privileged Daemons.app/Contents/Library/LaunchDaemons/` with root privileges, it's not recommended.
